lib/ansible/galaxy/api.py

```diff
@@ -454,9 +454,16 @@ def wait_import_task(self, task_url, timeout=0):
         start = time.time()
         wait = 2
 
+        # v3 returns relative task urls, v3 returns full urls
+        if 'v3' in self.available_api_versions:
+            parsed = urlparse(self.api_server)
+            n_url = parsed._replace(path=task_url).geturl()
```

Comment:
I've been meaning to ask... what's the deal with n_url?  I thought at first that it needed to be a native string for the APIs it's being used with but the more places I see it, the less that looks like the case.  Is it "normalized_url"?  If so, could you start using that?  We do not want to use `n_` or `b_` prefixes for anything other than variables containing native strings and variables containing byte strings.  It will defeat the purpose if we start doing this in other places.

Reply:
No idea what n_url is. It's just whatever the existing convention was. (Or at least, I don't remember it being my doing, but it wouldn't be the first time git blame has better memory than I do.)

Reply:
I also found `n_url` to be a bit confusing and collide with our string type prefixes for bytes, native, and unicode. It would be good to change `n_url` to something more descriptive, but probably in a separate PR.